### ISSUE TITLE
1.json explanation

### ISSUE THREAD

**Issue by junsukha** [dvlab-research/ControlNeXt#48]
Hi, thx for sharing the work!

I have some questions about `1.json`. 
The following is part of `1.json`.

`boxes` has four floats and `hands_boxes` has 8 floats (4 floats for each hand I guess). Are they representing top left and bottom right coordinates? If so, why do some `boxes` have floats bigger than 1.0? Aren't they normalized coordinates?

```json
"boxes": [
      [
        [
          0.2140564318056461,
          0.2613191584746043
        ],
        [
          0.6636727227105035,
          0.9934246321519214
        ]
      ]
    ],
    "hands_boxes": [
      [
        [
          0.28504847773799186,
          0.27063686450322466
        ],
        [
          0.3797045389811198,
          0.342516311009725
        ]
      ],
      [
        [
          0.30871249304877385,
          0.2892722765604656
        ],
        [
          0.3702389328568069,
          0.35050291617711393
        ]
      ]
    ],
```

An example of `boxes` that has larger float than 1.0
```json
"boxes": [
      [
        [
          0.2190996357688197,
          0.2629189019401868
        ],
        [
          0.664075881684268,
          1.097693155830105
        ]
      ]
```

# EDIT
The floats are indeed the coordinates of bboxes. They are used in the following way:

```python
# ControlNeXt-SVD-v2-Training/utils/pkl_dataset.py
                hands_mask = torch.zeros((guide_values.shape[0], 1, guide_values.shape[2], guide_values.shape[3]))
                if 'meta_info' in self.meta_info[idx].keys():
                    with open(self.meta_info[idx]['meta_info']) as f:
                        meta_info = json.load(f)
                    
                    for idx, frame_id in enumerate(frame_ids):
                        if 'hands_boxes' in meta_info[frame_id].keys():
                            hand_boxes = meta_info[frame_id]['hands_boxes']
                            for (x0, y0), (x1, y1) in hand_boxes:
                                draw_mask(hands_mask[idx], x0, y0, x1, y1, 1.)
```

**Comment by aleeyang** [comment]
same question, by the way, do you know  how to get hand box?

**Comment by junsukha** [comment]
@aleeyang check out `ControlNeXt-SVD-v2/dwpose/wholebody.py`. `inference_pose` outputs bboxes I guess. Haven't tried myself yet tho.

**Comment by aleeyang** [comment]
thank you. I am using the box generate from the hand keypoints.

**Comment by Pbihao** [comment]
We use DWPose to generate pose videos and meta-information, including the hands and body bounding boxes.

Are these boxes represented by the top-left and bottom-right coordinates?
Yes.

If so, why do some of the coordinates have values greater than 1.0?
As you may notice, this can occur when parts of the body are off-screen or partially hidden. Using a min function can resolve this. For more details, please refer to the DWPose project documentation.

Are the coordinates normalized?
Yes.
